backend/internal/supabase/migrations/20240917002108_base_schema.sql

```diff
@@ -60,4 +60,38 @@ create table if not exists review (
     comment text,
     created_at timestamp with time zone default now(),
     updated_at timestamp with time zone default now()
+);
+
```

Comment:
best practice is to handle new transactions in new migration script files, and i think supabase won't even let us reapply the same migration twice. super easy to make a new one though, [if you have the supabase cli you should be able to run](https://supabase.com/docs/guides/cli/local-development#database-migrations) `supabase migration new create_remaining_tables` within the `internal` folder and it'll autogenerate the timestamped file for you and then you can throw the new table statements in there
<img width="867" alt="image" src="https://github.com/user-attachments/assets/45dd24b0-fc5f-4a86-9ea3-d866fc3c3efe">